src/stories/atoms/Shape/Shape.test.tsx

```diff
@@ -0,0 +1,27 @@
+// Global Imports
+import React from 'react';
+import { composeStories } from '@storybook/testing-react';
+
+// Local Imports
+import { render } from 'utils/test-utils';
+import * as stories from 'stories/atoms/Shape/Shape.stories';
+
+beforeAll(() => {});
+afterEach(() => {});
+afterAll(() => {});
+
+
+const { Default } = composeStories(stories);
+
+describe('Shape unit test', () => {
```

Comment:
So testing the `Shape` component exposes the weakest part of this ticket as a teaching tool. It basically boils down to:

"Its not very intuitive to test visual components that don't have textual features that are apparent to the end user."

For example, its much easier to test the different stories for a component that is differentiated by its text content. 
```ts
// component.tsx

type Props {
 componentText: string
}
...rest
return(
  <span>{componentText}</span>
)
```

We can then write specific stories for the different text content and test those:

```ts
// Testing story `HelloWorld` which accepts `Hello World` as its `componentText` prop
const { getByText } = render(<HelloWorld />);
expect(getByText("Hello World")).toBeInTheDocument(); 

// Testing story `GoodbyeWorld` which accepts `Goodbye World` as its `componentText` prop
const { getByText } = render<GoodbyeWorld />);
expect(getByText("Goodbye World)).toBeInTheDocument();
```

And this idea actually forms the practical ethos for `React Testing Library`. It basically says that you should only test for things that are available to your end user. To put it another way, if the user can't see it  or interact with it, you shouldn't test for it. 

But to fully test `Shape` we need to break that rule. Look into the `Jest` [`toHaveAttribue`](https://github.com/testing-library/jest-dom#tohaveattribute) and see if you can find a way to test the `Rectangle` and `Color` stories.

Reply:
@perfect-perfect are you going to make the changes from my previous comment? If not its customary to say why and then to resolve the conversation.

Reply:
I made these changes. I went with another route, I found I was able to test the `Rectangle`  and `Color` stories by using the Jest `Matcher` `toHaveStyle`. My issue with using  `toHaveAttribute` was that at run-time `styled-components` assigns a hashed `class` to the components. This hashed `class` is then used in a runtime generated CSS file. Since the styling itself is not stored on the `<html>` elements `attributes` I found that using `toHaveStyle()` more directly tested our desired results.